src/components/estimatedScoreWidget/directives/estimatedScoreWidget.directive.js

```diff
@@ -58,16 +59,25 @@
                             isDiagnosticCompletedProm,
                             $q.when(false),
                             getSubjectOrderProm,
-                            getExamScoreProm
+                            getExamScoreProm,
+                            getUserGoalsCalcScore
                         ]).then(function (res) {
                             var estimatedScore = res[0];
                             var isDiagnosticCompleted = res[1];
                             var subjectOrder = res[3];
                             var examScoresFn = res[4];
+                            var userGoalsCalcScore = res[5];
 
                             scope.d.isDiagnosticComplete = isDiagnosticCompleted === 2;
 
-                            scope.d.userCompositeGoal = (userGoals) ? userGoals.totalScore : '-';
+                            if (userGoals) {
```

Comment:
Why is this needed?

If we need a different way of calculating the total score, then do it in the "UserGoalService" in the "getGoals()" func, so when you get the "userGoals" object it already contains the correct total score.....